### DISCUSSION TITLE
Compatible Device: UCAN STM32F072 chip

### DISCUSSION THREAD

**Discussion by Hye993** [collin80/SavvyCAN#812] [Q&A]
Question for all, is this device compatible with Savvycan? Want to run on windows, but don't want to spend big $$$

FYSETC UCAN Board Based on STM32F072 USB to CAN Adapter Support with CAnable / CandleLight / Klipper firmware
https://www.fysetc.com/products/fysetc-ucan-board-based-on-stm32f072-usb-to-can-adapter-support-with-canable-candlelight-klipper-firmware?_pos=1&_sid=5f00959b8&_ss=r

![FYSETC](https://github.com/collin80/SavvyCAN/assets/172888511/df9bc875-5d54-4f7d-81f9-c23c794d4f39)

**Comment by gedass2000** [comment]
Might be worth adding a few $$ and buying the insulated version ?

**Comment by mvrebolo** [comment]
I would like to know if this UCAN is compatible with SavvyCAN as well, if you got the answer, let me know, please
